storage/statedb/database.go

```diff
@@ -335,6 +345,18 @@ func NewDatabaseWithCache(diskDB database.DBManager, cacheSize int, daBlockNum u
 	}
 }
 
+// getTrieNodeCacheSizeMB retrieves size for trie cache
+func getTrieNodeCacheSizeMB() int {
+	totalPhysicalMem := float64(common.TotalPhysicalMemGB)
+	var cacheSizeGB float64
+	if totalPhysicalMem/4 < 2.5 { // 2.5GB margin at minimum (0.75GB for NewCache, 1.75GB for rest)
```

Comment:
```suggestion
	if totalPhysicalMem < 10 { // 2.5GB margin at minimum (0.75GB for NewCache, 1.75GB for rest)
```
This looks simpler, how about this?

Reply:
@winnie-byun As I told you yesterday , I like this better. :)
And please refine the comment also.